lib/zone-spec/fake-async-test.ts

```diff
@@ -32,13 +32,30 @@
   const OriginalDate = global.Date;
   class FakeDate {
     constructor() {
-      const d = new OriginalDate();
-      d.setTime(global.Date.now());
-      return d;
-    }
-
-    static UTC() {
-      return OriginalDate.UTC();
+      switch (arguments.length) {
+        case 0:
+          const d = new OriginalDate();
+          d.setTime(FakeDate.now());
+          return d;
+        case 1:
+          return new OriginalDate(arguments[0]);
```

Comment:
did you try `new OriginalDate(...arguments)` That would be nicer than having a big switch statemente.

Reply:
@mhevery , yes, this is a better way, I will change it , thank you.